product_docs/docs/pgd/5/scaling.mdx

```diff
@@ -18,32 +18,38 @@ your search_path, you need to schema-qualify the name of each function.
 
 ## Auto creation of partitions
 
-[`bdr.autopartition()`](/pgd/latest/reference/autopartition#bdrautopartition) creates or alters the definition of automatic range
-partitioning for a table. If no definition exists, it's created.
-Otherwise, later executions will alter the definition.
+PGD AutoPartition uses the [`bdr.autopartition()`](/pgd/latest/reference/autopartition#bdrautopartition)
+function to create or alter the definition of automatic range partitioning for a table. If
+no definition exists, it's created. Otherwise, later executions will alter the
+definition.
 
-[`bdr.autopartition()`](/pgd/latest/reference/autopartition#bdrautopartition) doesn't lock the actual table. It changes the
-definition of when and how new partition maintenance actions take place.
+The
+[`bdr.autopartition()`](/pgd/latest/reference/autopartition#bdrautopartition)
+function doesn't lock the actual table. It changes the definition of when and
+how new partition maintenance actions take place.
 
-[`bdr.autopartition()`](/pgd/latest/reference/autopartition#bdrautopartition) leverages the features that allow a partition to be
-attached or detached/dropped without locking the rest of the table
-(when the underlying Postgres version supports it).
+PGD AutoPartition leverages underlying Postgres features that allow a partition
+to be attached or detached/dropped without locking the rest of the table
+(Autopartion on PGD5 currently does not support this).
```

Comment:
Should we completely remove the first sentence instead of immediately saying that `PGD5 currently does not support this` ?

Reply:
Removed the section replaced with 

```
PGD Autopartition in PGD 5 currently locks the actual table while performing new partition maintenance operations.
```